alma/alma-installments-prestashop
# fix: Issue price widget in Cart Item

…nd handle if the price have coma in the class

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1912/[🧩-ps]-issue-widget-in-cart-item-send-the-static-price-of-the-product)

### Code changes


In the cart item we sent the wrong price.
And if the price more that 2 number after coma, the id was wrong to display the widget

### How to test

_As a reviewer, you are encouraged to test the PR locally._


Create a product with more that 2 number after coma and see this product in the cart item.
Now we will see the widget

### Checklist for authors and reviewers




### Non applicable
